packages/fxa-settings/src/pages/InlineTotpSetup/container.test.tsx

```diff
@@ -113,88 +123,91 @@ describe('InlineTotpSetupContainer', () => {
       );
     });
 
-    it('redirects when there is no account', () => {
-      render();
-      const location = mockLocationHook();
-      expect(mockNavigateHook).toHaveBeenCalledWith(
-        `/signup${location.search}`,
-        { state: undefined }
-      );
-    });
-
-    it('redirects when there is no session', () => {
-      mockAccountHook = () => ({ uid: 'quux' });
+    it('redirects when there is no signin state', () => {
       render();
-      const location = mockLocationHook();
+      const location = mockLocationHook(MOCK_QUERY_PARAMS, null);
       expect(mockNavigateHook).toHaveBeenCalledWith(
         `/signup${location.search}`,
         { state: undefined }
       );
     });
 
     it('redirects when the session is not verified', async () => {
-      mockAccountHook = () => ({ uid: 'quux' });
       mockSessionHook = () => ({ isSessionVerified: async () => false });
       render();
       const location = mockLocationHook();
       await waitFor(() => {
         expect(mockNavigateHook).toHaveBeenCalledWith(
           `/signin_token_code${location.search}`,
-          { state: undefined }
+          { state: MOCK_SIGNIN_LOCATION_STATE }
         );
       });
     });
 
     it('redirects when totp is active on the account', async () => {
-      mockAccountHook = () => ({
-        uid: 'quux',
-        refresh: () => {},
-        totpActive: true,
-      });
       mockSessionHook = () => ({ isSessionVerified: async () => true });
+      mockTotpStatusQuery.mockImplementation(() => {
+        return {
+          data: MOCK_TOTP_STATUS_VERIFIED,
+          loading: false,
+        };
+      });
+      jest
+        .spyOn(ApolloClientModule, 'useQuery')
+        .mockReturnValue(mockTotpStatusQuery());
       render();
       const location = mockLocationHook();
       await waitFor(() => {
         expect(mockNavigateHook).toHaveBeenCalledWith(
           `/signin_totp_code${location.search}`,
-          { state: undefined }
+          { state: MOCK_SIGNIN_LOCATION_STATE }
         );
       });
     });
   });
 
   describe('renders', () => {
-    const mockEmail = 'nomannisanislandexcepttheisleofmann@example.gg';
-
     beforeEach(() => {
-      mockAccountHook = () => ({
-        uid: 'quux',
-        email: mockEmail,
-        refresh: () => {},
-        totpActive: false,
-      });
       mockSessionHook = () => ({ isSessionVerified: async () => true });
     });
 
+    it.only('displays loading spinner when loading', async () => {
```

Comment:
I will remove this `.only` next time I push, I was checking to see where a warning was coming from after everything was passing.